src/Plugin/Block/ContactsEntity.php

```diff
@@ -163,6 +187,11 @@ public function getEditLink($mode) {
     }
 
     $params = $this->routeMatch->getRawParameters()->all();
+
+    if (empty($params['user']) || empty($params['subpage'])) {
```

Comment:
Why is this needed? If we don't have that, we shouldn't be rendering this block, but the DnD for it..?